server/controllers/CommentController.js

```diff
@@ -39,7 +52,7 @@ class CommentController {
         return response(res, 201, 'success', 'Comment created', null, getComment.dataValues);
```

Comment:
Nice work TJ!
I feel you should not need to getComment from the DB in order to confirm comment creation. You could just do
const commentCreate = await Comment.create({ id, content, userId, articleId });
and check
if(commentCreate)

Reply:
Yes comment created are gotten from the response body with getComment.datavalues

Reply:
<img width="1680" alt="screenshot 2019-01-18 at 5 47 38 am" src="https://user-images.githubusercontent.com/39820795/51366216-af742180-1ae4-11e9-9ee0-3f2ea100eeef.png">

Reply:
Great, Thanks!